util/dvsim/Testplan.py

```diff
@@ -527,8 +530,9 @@ def map_covergroups(self, cgs_found):
                 written += 1
 
         self.progress["Covergroups"] = {
-            "written": written,
             "total": total,
+            "written": written,
+            "passing": written,
             "progress": self._get_percentage(written, total),
```

Comment:
Here, progress is written/total? Is this intentional?

Reply:
perhaps passing/total is better?

Reply:
@imphil, it is indeed written / total. I am not sure I understand your concern. 
Covergroups do not have a concept of "passing", but the progress table does, so I just replicate the passing count with the written count. We are tracking coverage completion in a separate table anyway.

Reply:
is this to count passing tests?

In the PR description, looks like you want to only count passing test.
> The progress (%tage) is now computed as pass
percentage instead of written percentage. So if the test is written but
one or more seeds are failing, it will not be counted towards the
progress.

Reply:
Also, note that we do not currently have a mechanism to extract all of the "seen" covergroups from the VCS/Xcelium database, to check off against our testplan document. So covergroup info will not show up in this table at this time.

Reply:
Aah yes @weicaiyang. The progress captures passing tests AND written covergroups (not fully supported atm) in the same table. See line 506 above for passing test %tage. This line represents %tage of written covergroups.

Reply:
Got you. Thanks for the explanation.